unreleased.sh

```diff
@@ -2,6 +2,13 @@
 # Copyright (c) 2019 The DAML Authors. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+if [ "$#" -ne 1 ]; then
+    echo "Usage: ./unreleased.sh <revision range>"
+    echo "Prints all changelog entries added by the given revision range"
+    echo "For info about <revision range> please see gitrevisions(7)"
+    exit 64
```

Comment:
Why `64`?

Reply:
`#define EX_USAGE	64	/* command line usage error */`  in `sysexits.h`, not sure how useful that is in practice though.

Reply:
Sounds like solid reasoning to me.